modules/govuk_scripts/files/usr/local/bin/govuk_app_console

```diff
@@ -9,19 +9,25 @@
 
 set -eu
 
-if [ "$#" -ne "1" ] || [ "$1" = "-h" ]; then
-  echo "Usage: $(basename "$0") app_name" >&2
+if [[ ($# -ne 1 && $# -ne 2) || "$1" == "-h" ]]; then
+  echo "Usage: $(basename "$0") app_name [--sandbox]" >&2
   exit 1
 fi
 
 cd /var/apps/$1
 
 if [ -f bin/rails ]; then
-  sudo -u deploy govuk_setenv $1 bundle exec ./bin/rails console
+  sudo -u deploy govuk_setenv $1 bundle exec ./bin/rails console $2
 elif [ -f script/rails ]; then
-  sudo -u deploy govuk_setenv $1 bundle exec ./script/rails console
+  sudo -u deploy govuk_setenv $1 bundle exec ./script/rails console $2
 elif [ -f console ]; then
-  sudo -u deploy govuk_setenv $1 bundle exec ./console
+  if [[ -z $2 ]] ; then
```

Comment:
I took this magic from here: https://askubuntu.com/questions/444082/how-to-check-if-1-and-2-are-null